webui/tests/cypress/integration/02-notebooks.spec.ts

```diff
@@ -1,67 +0,0 @@
-describe('Notebooks List', () => {
-  before(() => {
-    cy.login();
-    cy.visit('/ui/notebooks');
-  });
-
-  describe('launching Notebooks', () => {
-    it('should say no notebooks have started yet', () => {
-      cy.get('.table .message').contains(/no notebooks have been started/i);
-    });
-
-    describe('launch button', () => {
-      it('should start a 1 slot notebook', () => {
-        cy.contains(/new notebook/i).click();
-        cy.get('.table tr').should('have.lengthOf', 1);
-      });
-
-      it('should present 0 slot launch button when clicking the dropdown', () => {
-        cy.get('button .fa-caret-down').click();
-        cy.contains(/cpu-only/i);
-        cy.get('button .fa-caret-down').click();
-      });
-
-      it('should launch a Notebook when clicking the cpu-only button', () => {
-        cy.get('button .fa-caret-down').click();
-        cy.contains(/launch new cpu/i).click();
-        cy.get('.table tr').should('have.lengthOf', 2);
-      });
-    });
-
-    describe('action buttons', () => {
-      it('there should be 3 action buttons', () => {
-        cy.get('tr:first-child td:last-child button').should('have.lengthOf', 3);
-      });
-
-      it('should open the logs modal when clicking the logs button', () => {
-        cy.get('tr:first-child').contains(/logs/i).click();
-        cy.get('.modal').contains(/logs for notebook/i);
-        cy.get('.modal .fa-times').click();
-      });
-
-      it('logs button should be enabled', () => {
-        cy.get('button').contains(/logs/i).closest('button').should('be.enabled');
-      });
-
-      it('open button should be enabled', () => {
-        cy.get('button').contains(/open/i).closest('button').should('be.enabled');
-      });
-
-      it('kill button should be enabled', () => {
-        cy.get('button').contains(/kill/i, { timeout: 30000 })
-          .closest('button').should('be.enabled');
-      });
-
-      it('kill should terminate the notebook', () => {
```

Comment:
we are removing/omiting this test (and a few other ones) here, is this intentional :thinking: ? can we keep them

Reply:
i moved the notebook specific tests into `02-tasks.spec.ts` and added tensorboard tests in there as well.